packages/react-client/src/__tests__/ReactFlight-test.js

```diff
@@ -468,6 +468,40 @@ describe('ReactFlight', () => {
       `);
   });
 
+  if (typeof FormData !== 'undefined') {
```

Comment:
We're actually testing in a version of Node that's too old for this in CI. We should bump to a newer version.